### PR TITLE
You can now buckle people to target stakes

### PR DESCRIPTION
**Pull request by silicons**
Target practice with your captured enemies!

### CODE REVIEW

**Comment by InsaneHyena** [comment]
How the fuck was this not a thing before.

**Comment by GunHog** [comment]
Those clown targets were just not cutting it for me. Time to use LIVE clowns!

**Comment by ShizCalev** [comment]
Changelog.

**Comment by AnturK** [comment]
Won't work without modifying density or buckling procs.

**Comment by arsserpentarium** [comment]
I need remove targets from circuitry lab map and put there cage with cliwn spawnpoint. Since you can use real clown for target practice

File: code/game/objects/structures/target_stake.dm [hunk 1]
```diff
@@ -28,6 +45,7 @@
 		T.pinnedLoc = src
 		T.density = TRUE
 		T.layer = OBJ_LAYER + 0.01
+		handle_density
```
Comment (lzimann) [inline code/game/objects/structures/target_stake.dm]: forgot `()`

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**